CMakeLists.txt

```diff
@@ -1216,14 +1213,12 @@ if(USE_CURL AND CURL_FOUND)
 
 endif()
 
-# Linux platforms may or may not have a separate libintl (may be a part of of libc)
-set(LIBINTL_AVAIL (LIBINTL_LIBRARY OR LINUX))
-check_deps(USE_GETTEXT LIBINTL_AVAIL GETTEXT_MSGFMT_EXECUTABLE)
```

Comment:
We use check_deps to check:

* Is the flag to enable compilation set to true? (Here USE_GETTEXT)
* Are the required dependencies for the target available? 

The value gets stored back into the first variable if the dependencies are missing. 

I think to get the same behavior you'll have to move the find_package()'s out from below and restore using check_deps.

Reply:
Well it's inside the `if (USE_GETTEXT)` block so the first one is already sorted. And `find_package` should handle the second one. So I think we're good?

Reply:
No, please re-read what I wrote. The way it is in this PR does not follow what is done elsewhere in regards to check_deps. The code in this PR checks for packages after testing for USE_GETTEXT, which will then fail the configuration process if the libraries aren't found. The way it's done elsewhere in the code base is it checks both "Can I?" (are the deps available) and "Should I?" (the USE_GETTEXT flag).

The build uses USE_GETTEXT set to true by default and therefore out-of-the-box will try to build with it and ignore if the deps aren't found. The user then also has the option to disable its use explicitly by setting USE_GETTEXT to false. Because Csound can build without gettext, this is the behavior we want out of the build system.

Reply:
Ok, I understand now. Sure, I can restore `check_deps`. Does seem a bit odd to allow users without `gettext` to build with `USE_GETTEXT` though.